app/services/albums.js

```diff
@@ -12,18 +12,15 @@ const options = {
 
 exports.getAlbums = id => {
   if (id) options.uri = `${apiUrl}/${id}`;
-  requestPromise(options)
-    .then(logger.info)
-    .catch(error => {
-      logger.error(error.message);
-    });
+  else options.uri = apiUrl;
```

Comment:
i think that you can do something like this:
 service:
```
exports.getAlbumSource = source => {
const options = {
  uri: `${apiUrl}source`,
  headers: {
    'User-Agent': 'Request-Promise'
  },
  json: true
};

return requestPromise(options).catch(error => {
  Promise.reject(externalApiError(error.message));
});
}
```

and in the controller for get album you can use:

```
const getAlbums = (req, res, next) => {
  albumsService
    .getAlbumSource(`/${req.params.id}`)
    .then(albums => res.status(200).send(albums))
    .catch(next);
};
```

and
```
const getAlbums = (req, res, next) => {
  albumsService
     .getAlbumSource(`/${req.params.id}/photos`)
    .then(albums => res.status(200).send(albums))
    .catch(next);
};
```
For get album photos.

by this way you only have one method in album service, because is the same call but with a small different in the uri